src/rpc/blockchain.cpp

```diff
@@ -1185,9 +1185,9 @@ static RPCHelpMan verifychain()
     return RPCHelpMan{"verifychain",
                 "\nVerifies blockchain database.\n",
                 {
-                    {"checklevel", RPCArg::Type::NUM, /* default */ strprintf("%d, range=0-4", DEFAULT_CHECKLEVEL),
```

Comment:
Would it make sense to attempt to make this a scripted-diff? 

It looks like one can change `/* default */` to `/* default-hint */` in one commit (by hand) and then write a sed script that does the replacements for you. It would probably be hard unless done in a partial way. Just thinking out loud.